server/service/campaigns_test.go

```diff
@@ -128,21 +141,22 @@ func TestLiveQueryAuth(t *testing.T) {
 			false,
 			false,
 		},
-		// NOTE: this specific case is not covered by the rego authorization policy,
-		// it is at the datastore level that a filter is applied to only consider
-		// hosts that the user can see (that is, a fleet.TeamFilter is passed to
-		// ds.HostIDsInTargets and that call applies the filter to return only
-		// allowed hosts).
-		/*
-			{
-				"team admin, target not set to own team",
-				&fleet.User{Teams: []fleet.UserTeam{{Team: fleet.Team{ID: 1}, Role: fleet.RoleAdmin}}},
-				ptr.Uint(2),
-				false,
-				false,
-				true,
-			},
-		*/
+		{
+			"team admin, no team target",
+			&fleet.User{Teams: []fleet.UserTeam{{Team: fleet.Team{ID: 1}, Role: fleet.RoleAdmin}}},
+			nil,
+			false,
+			false,
+			false,
+		},
+		{
+			"team admin, target not set to own team",
+			&fleet.User{Teams: []fleet.UserTeam{{Team: fleet.Team{ID: 1}, Role: fleet.RoleAdmin}}},
+			ptr.Uint(2),
+			false,
```

Comment:
It would fail on the `ActionRun` check, but the reason it's set to not fail here is due to the complexity of testing RunNew (documented in a comment below).